Sources/ApodiniDelta/Schema/Schema.swift

```diff
@@ -0,0 +1,140 @@
+//
```

Comment:
I do not think that it would make sense currently to move it there. There is some redundancy happening there currently, between my `Schema`, `ProtobufferMessage` and `JSONSchema` of `OpenAPIKit` used in `OpenApiIE`. All these types rely on `ReflectionInfo` and extract the infos used in their exporter. That's what I am also doing. But for sure, I will discuss this issue with Max